core/src/main/scala/org/apache/spark/sql/delta/commands/VacuumCommand.scala

```diff
@@ -132,6 +132,12 @@ object VacuumCommand extends VacuumCommandImpl with Serializable {
       val relativizeIgnoreError =
         spark.sessionState.conf.getConf(DeltaSQLConf.DELTA_VACUUM_RELATIVIZE_IGNORE_ERROR)
 
+      val fileListingParallelism =
+        spark.sessionState.conf.getConf(DeltaSQLConf.DELTA_VACUUM_FILE_LISTING_PARALLELISM)
+          .getOrElse(
+            Option(spark.sessionState.conf.parallelPartitionDiscoveryParallelism).getOrElse(200)
+          )
```

Comment:
Could you add unit tests for this config behavior? `spark.sessionState.conf.parallelPartitionDiscoveryParallelism` will never return null if I recall correctly.

Reply:
`spark.sessionState.conf.parallelPartitionDiscoveryParallelism` is a spark conf. Not sure how can I check if it is set or not. I can compare the value to its default one 10000. but if the default changes in spark, we will use always the default instead of 200.